### PR TITLE
Codecoverage fix

### LINKED ISSUE #3111: Code Coverage not working in 2.2.0-RC2
Hello,

In 2.2.0-RC2 version even with 4.8 or 5.3 phpunit I get wrong coverage. I don't get any error but coverage is wrong. I use a local vagrant box with php7.

Is it something known or I am doing something wrong?

I use it to test symfony app and with 2.1 version works better (well not 100%% accurate) but when I change to 2.2 I get all the time:

Summary:
Classes: 5.56% (3/54)
Methods: 0.00% (0/248)
Lines: 0.00% (0/64889)

I test locally in Vagrant nginx- php7-mariadb machine. Here is is my codeception.yml:

    actor: Tester
    coverage:
        enabled: true
        include:
          - src/*
     paths:
        tests: tests
        log: tests/_output
        data: tests/_data
        support: tests/_support
        envs: tests/_envs
    settings:
        bootstrap: _bootstrap.php
        colors: true
        memory_limit: 1024M
    extensions:
       enabled:
            - Codeception\Extension\RunFailed

### PR DESCRIPTION
**Pull request by DavertMik**
Added fix for Codecoverage collection in Cept and Cest formats. Fixes #3111

### CODE REVIEW

**Comment by Naktibalda** [comment]
Gherkin and Test formats too?

**Comment by DavertMik** [comment]
Gherkin too, test format was not affected
1 черв. 2016 р. 11:57 "Gintautas Miselis" <notifications@github.com> пише:

File: src/Codeception/Test/Feature/CodeCoverage.php [hunk 1]
```diff
@@ -23,11 +25,9 @@ public function codeCoverageEnd($status, $time)
         if (!$codeCoverage) {
             return;
         }
-        $linesToBeCovered = \PHPUnit_Util_Test::getLinesToBeCovered(get_class($this->testClassInstance), 'test');
-        $linesToBeUsed = \PHPUnit_Util_Test::getLinesToBeUsed(get_class($this->testClassInstance), 'test');
 
         try {
-            $codeCoverage->stop(true, $linesToBeCovered, $linesToBeUsed);
```
Comment (Naktibalda) [inline src/Codeception/Test/Feature/CodeCoverage.php]: Are we losing anything by not providing $linesToBeCovered and $linesToBeUsed?

Comment (DavertMik) [inline src/Codeception/Test/Feature/CodeCoverage.php]: Those values are taken from annotations like `@covers`. I don't think they can be implemented for multiple formats, and they were not documented to be used inside Cept or Cest files. So for now they are just ignored. Probably in future we can come to idea how to get it to use

**Pull request merged**